### PR TITLE
Fix matrix comparison in sfTransform_equal

### LINKED ISSUE #166: Graphics/Transform.cpp:167:26: warning: comparison between two arrays
Recent gcc compiler can be made to say:

CSFML-2.5.1/src/SFML/Graphics/Transform.cpp:167:26: warning: comparison between two arrays [-Warray-compare]

Source code is

   return (left->matrix == right->matrix) ? sfTrue : sfFalse;

### PR DESCRIPTION
**Pull request by Bqleine**
Fixes #166

### CODE REVIEW

**Review by eXpl0it3r — APPROVED** [review]

**Comment by eXpl0it3r** [comment]
Thank you for the fix!

File: src/SFML/Graphics/Transform.cpp [hunk 1]
```diff
@@ -164,6 +164,9 @@ sfBool sfTransform_equal(sfTransform* left, sfTransform* right)
     CSFML_CHECK_RETURN(left, false);
     CSFML_CHECK_RETURN(right, false);
 
-    return (left->matrix == right->matrix) ? sfTrue : sfFalse;
+    for (int i = 0; i < 9; ++i) {
+        if (left->matrix[i] != left->matrix[i]) return sfFalse;
```
Comment (ChrisThrasher) [inline src/SFML/Graphics/Transform.cpp:168]: This is just comparing the left vector to itself

Comment (eXpl0it3r) [reply src/SFML/Graphics/Transform.cpp:168]: Woops, looks like a copy paste issue, should be `right->` on the right side of course!

Comment (ChrisThrasher) [reply src/SFML/Graphics/Transform.cpp:168]: I'm fixing this in an upcoming PR

Comment (Bqleine) [reply src/SFML/Graphics/Transform.cpp:168]: Oops 😅

**Pull request merged**